src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/PersonAliasEntity.kt

```diff
@@ -42,5 +43,27 @@ class PersonAliasEntity(
 
   @Version
   var version: Int = 0,
+) {
+  companion object {
+    private fun from(personAlias: PersonAlias): PersonAliasEntity? {
+      return if (isAliasPresent(personAlias.firstName, personAlias.middleName, personAlias.lastName)) {
+        PersonAliasEntity(
+          firstName = personAlias.firstName,
+          middleNames = personAlias.middleName,
+          lastname = personAlias.lastName,
+        )
+      } else {
+        null
+      }
```

Comment:
This is fine but as a minor style point I quite like `when` for these conditions - it means you have to cover every possibility if(when) :-) it expands